examples/pipelines/finetune_stable_diffusion/components/embedding_component/src/main.py

```diff
@@ -51,6 +51,8 @@ class EmbeddingComponent(HFDatasetsTransformComponent):
     """
     Component that embeds the images using a CLIP model from Hugging Face.
     """
+    data_sources_in = [("images", express.Image)]
```

Comment:
I think we should define this as config somewhere, so this is easily accessible for other classes and external tools.